app/questionnaire/questionnaire_schema.py

```diff
@@ -116,6 +121,49 @@ def placeholder_transform_section_dependencies_by_block(
     def answer_dependencies(self) -> ImmutableDict[str, set[AnswerDependent]]:
         return ImmutableDict(self._answer_dependencies_map)
 
+    @cached_property
+    def min_and_max_map(self) -> Any:
+        return make_immutable(self._min_and_max_map)
+
+    def _create_min_max_map(
```

Comment:
It doesn't look like we are taking into account decimals.

If min was 0, but decimal was 2, right now it would return min length as 1 but it should be 3 as it would be `0.00`. Same applies to max.